cmake/libdatachannel-config.cmake.in

```diff
@@ -0,0 +1,19 @@
+@PACKAGE_INIT@
+
+include(CMakeFindDependencyMacro)
+find_dependency(Threads)
+
+if (@USE_GNUTLS@)
+    find_dependency(GnuTLS)
+else()
+    find_dependency(OpenSSL)
+endif()
+
+#if (@USE_NICE@)
+#    find_dependency(LibNice)
+#else()
+#    find_dependency(LibJuice)
+#endif()
```

Comment:
Why is this commented out ? libjuice is always a submodule for now so I guess only libnice should be considered.

Also, what about libSRTP?

Reply:
Temporarily comment because it does not export configuration files.